Raspi/Test/mock_wireless_module.py

```diff
@@ -0,0 +1,170 @@
+# MQTT wireless sensor test script that sends fake randomly generated data
+# To do a single test --> python3 mqtt_wireless_test.py -t 0
+
+import sys
+import time
+import random
+import argparse
+import json
+import paho.mqtt.client as mqtt
+
+class Sensors:
```

Comment:
Good start on writing a class! However, I would move this out to it's own file where the base class is called `MockSensor` and then you have the other Sensors (CO2, humidity etc.) inherit from this base class.